website/src/templates/index.js

```diff
@@ -25,6 +25,11 @@ const AlertSpace = ({ nightly, legacy }) => {
     const isOnline = useOnlineStatus()
     return (
         <>
+            {isOnline && (
+                <Alert title="💥 We'd love to learn more about your experience with spaCy!">
+                    <Link to="https://form.typeform.com/to/aMel9q9f">Take our survey here.</Link>
```

Comment:
```suggestion
                    <Link to="https://form.typeform.com/to/aMel9q9f">Take our 2023 survey here.</Link>
```
Maybe this makes it clear this is a current, limited opportunity?

Reply:
To me, this kind of makes it feel like it will be available for the rest of the year. I'm not sure of a better option though

Reply:
Oh yea, that's a good point. Hm. Maybe just leave as is then? We'd likely run this for about a month, until April 10th.

Reply:
I would say leave as is, or change to something like `<LINK>Take our survey here</LINK> until April 10th`. I think I prefer as is though.